components/PickerBase/PickerBase.vue

```diff
@@ -23,14 +23,17 @@
               >*</sup
             >
           </div>
-          <BButton
-            v-if="showAllButton"
-            id="picker-all-button"
-            data-cy="picker-all-button"
-            size="sm"
-            variant="primary"
-            v-on:click="pickAll()"
-            >All</BButton
+         <BButton
+           v-if="showAllButton"
+           id="picker-all-button"
+           data-cy="picker-all-button"
+           size="sm"
+           :variant="checked.length === options.length ? 'danger' : 'primary'"
```

Comment:
The `danger` and `primary` variants seem a bit overly intrusive to me.  I would just leave this as `primary` and let the icon be sufficient for indicating what will happen. The PickListBase doesn't use the red and green variants and I prefer it.